### DISCUSSION TITLE
Typo inside [@types/smartystreets-javascript-sdk]

### DISCUSSION THREAD

**Discussion by maritzuhcom** [DefinitelyTyped/DefinitelyTyped#60752] [Issues with a @types package]
Hello, in the types file [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/11436c5a19fc6aabfd6b8d93b37dac38b4ab9bc2/types/smartystreets-javascript-sdk/index.d.ts#L219) it should read `addressee`, not `adressee`

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60861
